pnpm-lock.yaml

```diff
@@ -14202,6 +14440,45 @@ packages:
       - esbuild
       - uglify-js
 
+  /webpack@5.88.0:
```

Comment:
Where did this come from?

Reply:
Looks like it came via fixtures 

https://github.com/seek-oss/sku/blob/8d11309848dc183184099578af4b95cd5a99edb1/pnpm-lock.yaml#L344-L354

Reply:
I ran a `pnpm dedupe` and it removed some of them, but there's still two of them.

Reply:
[`e9983f9` (#844)](https://github.com/seek-oss/sku/pull/844/commits/e9983f9e0cc6be17cff6065a07489f2f103acfa5)